src/DynamoCoreWpf/Views/Core/NodeView.xaml

```diff
@@ -5,7 +5,6 @@
              xmlns:dynui="clr-namespace:Dynamo.UI.Controls"
              xmlns:p="clr-namespace:Dynamo.Wpf.Properties"
              xmlns:configuration="clr-namespace:Dynamo.Configuration;assembly=DynamoCore"
-             xmlns:fa="http://schemas.fontawesome.io/icons/"
```

Comment:
might this negatively effect nodeViews which reference font awesome?

Reply:
Shouldn't be the case, at least this name space is not referenced at all.. I can also remove if there are known issues but I can't think of any side effect